core/modules/utils/filesystem.js

```diff
@@ -491,20 +524,20 @@ exports.cleanupTiddlerFiles = function(options, callback) {
 	bootInfo = options.bootInfo || {},
 	title = options.title || "undefined";
 	if(adaptorInfo.filepath && bootInfo.filepath && adaptorInfo.filepath !== bootInfo.filepath) {
-		return $tw.utils.deleteTiddlerFile(adaptorInfo, function(err){
+		$tw.utils.deleteTiddlerFile(adaptorInfo, function(err){
 			if(err) {
 				if ((err.code == "EPERM" || err.code == "EACCES") && err.syscall == "unlink") {
 					// Error deleting the previous file on disk, should fail gracefully
-					$tw.syncer.displayError("Server desynchronized. Error cleaning up previous file for tiddler: "+title, err);
-					return callback(null);
+					$tw.syncer.displayError("Server desynchronized. Error cleaning up previous file for '"+title+"'", err);
```

Comment:
I think, the existing error message, makes it more obviour, that the `title` is dynamic. So I would prefer:

```
$tw.syncer.displayError("Server desynchronized. Error cleaning up previous file for tiddler: \""+title+"\"", err);
```